src/style.css

```diff
@@ -1 +1,86 @@
 /* Styles go here */
+:root {
+  --primary-black: #000;
+  --secondary-light-blue: #00acdc;
+}
+
+* {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+a {
+  text-decoration: none;
+
+}
+
+li {
+  list-style: none;
+}
```

Comment:
I'd move these rules to appropriate selectors
but not to apply to the whole page